tests/rptest/tests/rpk_config_test.py

```diff
@@ -0,0 +1,183 @@
+# Copyright 2021 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from ducktape.utils.util import wait_until
+
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.clients.rpk_remote import RpkRemoteTool
+from rptest.services.rpk_consumer import RpkConsumer
+
+import yaml
+
+import random
+
+
+class RpkConfigTest(RedpandaTest):
```

Comment:
it's not clear to me why this is a ducktape test. does it really depend on a redpanda cluster?

Reply:
It has many basic tests, yes, but I expect more tests like [this one](https://github.com/vectorizedio/redpanda/pull/985/files/ce29cd2f53da3de0dacb8c6a034461ecc9fa5ea3#diff-ab954ab6eb4a080a63c52d284ffab5bc004b377466e363e4543a4d0184df5e5eR173-R183) will be added.

Reply:
keep in mind that each test spins up a full test cluster. so if the test doesn't require a test cluster, it can live outside of ducktape

Reply:
Sorry, I'd forgotten to send this:

I think it's still worth having these tests run in a realer setup, since unit tests are in a very controlled context (mocks, in-memory FS, etc). Side-effects may arise and go unnoticed.